rpc/jsonrpc/server/http_server.go

```diff
@@ -261,7 +253,7 @@ func (h maxBytesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // Listen starts a new net.Listener on the given address.
 // It returns an error if the address is invalid or the call to Listen() fails.
-func Listen(addr string, config *Config) (listener net.Listener, err error) {
+func Listen(addr string, maxOpenConnections int) (listener net.Listener, err error) {
```

Comment:
Realizing this will be API breaking, will remove. The goal here was to pass only what is actually used by the function so that it's clear what data the caller needs to pass through. This should be strongly preferred where possible imo to make using our APIs safe and easy and turn possible runtime bugs (field not set) into compile time bugs (function call missing param).

Reply:
I agree with the reasoning and from my standpoint, I think it's fine to break the API of this function. I do think it's possible that this function is getting used but it is a relatively simple change and there will be a considerable amount of work coming up anyway for developers that use tendemrint as a library

Reply:
agree, lets break it.